mobile/native/src/ln_dlc/recover_rollover.rs

```diff
@@ -1,74 +1,85 @@
-            .find(|dlc_channel| dlc_channel.channel_id == channel_details.channel_id);
-
-        let dlc_channel = match dlc_channel {
-            Some(dlc_channel) => dlc_channel,
+            .find(|dlc_channel| dlc_channel.channel_id == channel_details.channel_id)
+        {
+            Some(subchannel) => subchannel,
             None => {
-                tracing::debug!("No dlc channel found. All good!");
                 return Ok(());
             }
         };
 
-        let dlc_channel_id = match dlc_channel.get_dlc_channel_id(0) {
+        let channel_id_hex = subchannel.channel_id.to_hex();
+
+        let dlc_channel_id = match subchannel.get_dlc_channel_id(0) {
             Some(dlc_channel_id) => dlc_channel_id,
             None => {
-                tracing::warn!(channel_id=%hex::encode(dlc_channel.channel_id), "Couldn't get a dlc channel id for a dlc channel");
+                tracing::debug!(
+                    channel_id=%channel_id_hex,
+                    "Cannot consider subchannel for rollover recovery without DLC channel ID"
+                );
                 return Ok(());
             }
         };
 
-        let channel = self
+        let dlc_channel_id_hex = dlc_channel_id.to_hex();
+
+        let dlc_channel = self
             .inner
             .dlc_manager
             .get_store()
             .get_channel(&dlc_channel_id)?;
 
-        let signed_channel = match channel {
+        let signed_channel = match dlc_channel {
             Some(Channel::Signed(signed_channel)) => signed_channel,
-            Some(channel) => {
-                tracing::warn!(dlc_channel_id=%hex::encode(dlc_channel_id), "Found channel in unexpected state. Expected: Signed, Found: {channel:?}");
+            Some(_) => {
                 return Ok(());
             }
             None => {
-                tracing::warn!(dlc_channel_id=%hex::encode(dlc_channel_id), "Couldn't find channel");
+                tracing::warn!(
+                    channel_id=%channel_id_hex,
+                    dlc_channel_id=%dlc_channel_id_hex,
+                    "Expected DLC channel associated with subchannel not found"
+                );
                 return Ok(());
             }
         };
 
-        match signed_channel.state {
-            SignedChannelState::RenewOffered { .. }
-            | SignedChannelState::RenewAccepted { .. }
-            | SignedChannelState::RenewConfirmed { .. }
-            | SignedChannelState::RenewFinalized { .. } => {
-                let state = ln_dlc_node::node::signed_channel_state_name(&signed_channel);
-                tracing::warn!(state, "Found signed channel contract in an intermediate state. Rolling back, expecting coordinator to retry rollover!");
-                self.inner.rollback_channel(&signed_channel)?;
-            }
-            _ => {
-                tracing::debug!(signed_channel_state=%signed_channel.state, "Channel is not in an intermediate rollover state. All good.")
```

Comment:
For debugging it would be helpful to have that log message.

Reply:
I deleted these logs because I thought they were a bit noisy when debugging locally. But I can add them again.

Reply:
I have reintroduced all of them, but edited them slightly.